lib/datarockets/style/cop/layout/array_alignment_extended.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
```

Comment:
You can implement your cops in RuboCop::Cop namespace - it will help to move your cop to `rubocop` repo easier and inherit your cop from `Cop` instead of `RuboCop::Cop::Cop`

Reply:
Hmm, I see files structure of our project and look:
in rubocop rep:
```
/rubocop/formatter
/rubocop/version.rb
/rubocop.rb
```
in our rep:
```
/style/formatter
/style/version.rb
/style.rb
```

So looks like `/rubocop/cop` should be placed in `/style/cop` (`Datarockets::Style::Cop`)

Thoughts?